CRM/MembershipExtras/Test/Fabricator/Base.php

```diff
@@ -33,10 +33,29 @@ public static function fabricate(array $params = []) {
       throw new \Exception('Entity name cannot be empty!');
     }
 
-    $params = array_merge(static::$defaultParams, $params);
-    $result = civicrm_api3(static::$entityName, 'create', $params);
+    $defaultParams = static::getDefaultParameters();
```

Comment:
you will need to re-write your tests according to the latest changes in the master branch.